pkg/kubernetes/scripts/virtual-machines/kube-middleware.jsx

```diff
@@ -49,4 +51,10 @@ async function vmDelete({ vm }) {
     return kubeMethods.delete(selfLink); // no value is returned (empty promise). An exception is thrown in case of failure
 }
 
-export { vmDelete, vmCreate };
+async function fetchStatSummary(nodeName) {
```

Comment:
I think we should avoid ES7 for now. 
IIUC, this is the first time ES7 is used in this project and I'm not sure if recent babel configuration is ready for it.
Otherwise, this would need to be properly tested in all supported browsers and their old versions.

Reply:
`async` is already used in the delete method. Anyway, I think this should be resolved by `babel-plugin-transform-async-to-generator`

Reply:
Right, I forgot I've already added that :-) I'm sorry for the fuss.